src/OpenTK.GraphicsLibraryFramework/Enums/InitHint.cs

```diff
@@ -9,10 +9,32 @@
 
 namespace OpenToolkit.GraphicsLibraryFramework
 {
+    /// <summary>
+    /// Initialization hints are set before <see cref="IGLFW.Init"/> and affect how the library behaves until termination.
+    /// Hints are set with <see cref="IGLFW.InitHint"/>.
+    /// </summary>
     public enum InitHint
     {
+        /// <summary>
+        /// Used to specify whether to also expose joystick hats as buttons,
+        /// for compatibility with earlier versions of GLFW that did not have
+        /// <see cref="IGLFW.GetJoystickHats"/>.
+        /// Set this with <see cref="IGLFW.InitHint"/>.
+        /// </summary>
         JoystickHatButtons = 0x00050001,
+
+        /// <summary>
+        /// Used to specify whether to set the current directory to the application to the Contents/Resources
+        /// subdirectory of the application's bundle, if present.
+        /// Set this with <see cref="IGLFW.InitHint"/>.
+        /// </summary>
         CocoaChdirResources = 0x00051001,
```

Comment:
Docs for this and CocoaMenubar should specify that these do absolutely nothing on platforms other than macOS.

Reply:
Used GLFW original documentation here as well.

Should I add it inside of a "remarks" tag?

Reply:
Yeah, that makes sense. Something like "Only effects macOS; no effect on other platforms."